apps/vaporgui/VLineEdit.h

```diff
@@ -0,0 +1,129 @@
+#pragma once
+
+#include <string>
+
+#include <QWidget>
+#include <QLabel>
+#include <QLineEdit>
+#include <QSpinBox>
+#include <QCheckBox>
+#include <QWidgetAction>
+
+#include "VContainer.h"
+
+//! class VLineEdit
+//! Wraps a QLineEdit with vaporgui's standard setter/getter functions.
+//! Handles string and double types, as well as precision and display of
+//! double values.
+
+class VLineEdit : public VContainer {
+    Q_OBJECT
+
+public:
+    VLineEdit( const std::string& value = "");
+
+    ~VLineEdit();
```

Comment:
If you need a manual destructor, you should implement rule of 5.

Reply:
Thanks.  Fixed.